test/integration_test.rb

```diff
@@ -0,0 +1,62 @@
+require 'helpers/kubeclient_helper'
```

Comment:
Hmm, I get your point that it isn't a test to run, but it really doesn't fit with the standard definition of a helper either (i.e. a module full of related helper methods). And `SomeClassTest < IntegrationTestHelper` feels pretty wrong to me. I tried looking at what rails does, which is how I landed on this name (it's not actually under `test/` though):
- rails/activesupport/lib/active_support/test_case.rb
- rails/activerecord/test/cases/integration_test.rb

I'm not really sure what's best. I do kinda like `KubernetesDeploy::IntegrationTest < KubernetesDeploy::TestCase < ::Minitest::Test`, which I think implies `test/integration_test.rb` and `test/test_case.rb`?